app/services/ClaimToAdjustService.scala

```diff
@@ -89,20 +88,47 @@ class ClaimToAdjustService @Inject()(val financialDetailsConnector: FinancialDet
     }).map(_.flatten)
   }
 
-  def getPoaTaxYearForEntryPoint(nino: Nino)(implicit hc: HeaderCarrier): Future[Either[Throwable, Option[TaxYear]]] = {
+  def getPoaTaxYearForEntryPoint(nino: Nino)(implicit hc: HeaderCarrier): Future[Either[Throwable, Option[PaymentOnAccount]]] = {
```

Comment:
if we can keep original function and have a second function with the name:
getPoaTaxYearForEntryPointAsPaymentOnAccount or similar